hnn_core/tests/test_gui.py

```diff
@@ -563,3 +563,41 @@ def add_child_decorated(self, to_add):
     # Check if the widgets are relinked, the selected index should be synced
     gui.tab_group_1.selected_index = 0
     assert gui.tab_group_2.selected_index == 0
+
+
+def test_gui_download_simulation():
+    """Test the GUI download simulation pipeline."""
+    gui = HNNGUI()
+    _ = gui.compose()
+    gui.params['N_pyr_x'] = 3
+    gui.params['N_pyr_y'] = 3
+
+    # Run a simulation with 3 trials
+    gui.widget_dt.value = 0.85
+    gui.widget_ntrials.value = 3
+
+    # set synch inputs to first driver in simulation
```

Comment:
Seems like a legacy comment from copying the another test.

Reply:
Done